src/Hamiltonians/HubbardReal1D.jl

```diff
@@ -4,7 +4,8 @@
 Implements a one-dimensional Bose Hubbard chain in real space.
 
 ```math
-\\hat{H} = -t \\sum_{\\langle i,j\\rangle} a_i^† a_j + \\frac{u}{2}\\sum_i n_i (n_i-1)
+\\hat{H} = - \\sum_i \\left(t^* a_i^† a_{i+1} + t a_i a_{i+1}^†\\right) + 
+\\frac{u}{2}\\sum_i n_i (n_i-1)
```

Comment:
I'd prefer to write it like this:
```suggestion
\\hat{H} = - \\sum_i \\left(t a_i^† a_{i+1} + \\mathrm{h.c.}\\right) + 
\\frac{u}{2}\\sum_i n_i (n_i-1)
```
Alternatively (and maybe a bit clearer and more explicit):
```
\\hat{H} = - \\sum_i \\left(t a_i^† a_{i+1} + t^* a_{i+1}^† a_i \\right) + 
\\frac{u}{2}\\sum_i n_i (n_i-1)
```
This will require changing the code as well, as the hop to the right gets the complex conjugate amplitude.

The same for the other 1D Hubbard models.